cg/meta/upload/fohm/fohm.py

```diff
@@ -268,3 +281,12 @@ def update_uploaded_at(self, case_id: str) -> None:
         case_obj: models.Family = self.status_db.family(case_id)
         case_obj.analyses[0].uploaded_at = dt.datetime.now()
         self.status_db.commit()
+
+    def delete_empty_directories(self):
+        """Delete any empty directories in the fohm directory."""
+        fohm_root = self._daily_bundle_path.parent.absolute().as_posix()
+        directories = list(os.walk(fohm_root))
```

Comment:
Would be nice to use pathlib instead of os if possible

Reply:
I could not figure out a clean way to do the same thing with pathlib